src/Persistence.LinnApps/Repositories/BomVerificationHistoryRepository.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace Linn.Purchasing.Persistence.LinnApps.Repositories
+{
+    using System;
+    using System.Linq;
+    using System.Linq.Expressions;
+
+    using Linn.Common.Persistence;
+    using Linn.Purchasing.Domain.LinnApps.Boms;
+
+    using Microsoft.EntityFrameworkCore;
+
+    public class BomVerificationHistoryRepository : IRepository<BomVerificationHistory, int>
```

Comment:
you should inherit EntityFrameworkRepository and only overwrite the methods you need to change (the ones with includes)